templates/common/run/diff_test.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2024 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package run
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+
+	abctestutil "github.com/abcxyz/abc/templates/testutil"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestDiff(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name        string
+		dirContents map[string]string
+		color       bool
+		// file1Contents string
+		// file2Contents string
```

Comment:
Curious if these two will have some usecase in the future.

Reply:
Just fixed :)